src/Microsoft.ML.Data/Transforms/NormalizeUtils.cs

```diff
@@ -52,17 +52,19 @@ public interface IColumnAggregator<T>
         void Finish();
     }
 
-    internal interface IColumnFunction : ICanSaveModel
+    public abstract class IColumnFunction : ICanSaveModel
```

Comment:
>IColumnFunction [](start = 26, length = 15)

This is a bad name for a class :)

Reply:
I suggest `NormalizerModelParametersBase`, and then derived would be `AffineModelParameters` etc.

---
In reply to: [232814149](https://github.com/dotnet/machinelearning/pull/1594#discussion_r232814149) [](ancestors = 232814149)